### ISSUE TITLE
Deep Copy Deep Paste places AI at set random location

### ISSUE THREAD

**Issue by BedPleyar** [zen-mod/ZEN#327]
**Arma 3 Version:** `1.96.146114` (stable)
**CBA Version:** `3.14.0.200207` (stable)
**ZEN Version:** `1.6.1.14` (stable)

**Mods:**
```
- CBA_A3
- ZEN
- RHS
- CUP
- 3CB
- The list goes on, mostly just weapon/uniform/veh mods
```

**Description:**
When deep-copying and deep-pasting units in Zeus, the AI will be pasted at a set location (not where you are pointing) on the map.
The AI should be pasted where you are pointing in Zeus.

**Steps to reproduce:**
Run mission on dedicated multiplayer server, deep copy and deep paste.

**Where did the issue occur?**
- Dedicated Multiplayer 

**Additional information:**
It works in singleplayer but not multiplayer.

**RPT log file:**
N/A

**Screenshots:**
AI to be deep copy-pasted:
![Capture](https://user-images.githubusercontent.com/62317472/76918839-2bbbed80-68bf-11ea-82b1-64be388ab55d.PNG)
AI location on map after paste, original location is on the left:
![capp](https://user-images.githubusercontent.com/62317472/76918923-53ab5100-68bf-11ea-8743-418656c9d79d.PNG)
AI in water:
![st](https://user-images.githubusercontent.com/62317472/76918958-6756b780-68bf-11ea-9c70-acf4a7e3573f.PNG)

**Comment by mharis001** [comment]
Are you sure you are using this correctly. Like vanilla copy/paste, deep copy/paste places objects with the same offset from the mouse position that they were copied with. For example, if the objects are 100 m to the right of the mouse position when copied, then they will be placed 100 m to the right of the mouse position when pasted.

**Comment by BedPleyar** [comment]
@mharis001 I am absolutely sure, normal copy/paste works in the same environment that the deep copy/paste doesn't work in.

To elaborate further, regardless of where the cursor is when copying the unit, deep copy-paste **always** places the unit at a set location on the map.

Referencing the images in the original post, no matter where my cursor is when copying those cop units, (1st image) deep copy-paste will **always**  paste them relative to that random location in the gulf (2nd image).

**Comment by mharis001** [comment]
You are using 1.6.1, does this issue still occur with 1.7.0? An issue with deep copy/paste positioning with multiple curators was fixed in 1.7.0.

**Comment by mharis001** [comment]
I think this issue was fixed in 1.7.0. If it still occurs, please reopen.
